gammapy/datasets/make.py

```diff
@@ -64,3 +70,90 @@ def sigma_energy_theta(energy, theta, sigma):
                                        zenith=zenith_hi)
 
     return psf
+
+
+def generate_observation_table(observatory, n_obs):
```

Comment:
I would prefer `make_test_observation_table`.
For consistency I prefer `make` over `generate` (see [here](https://gammapy.readthedocs.org/en/latest/datasets/index.html#introduction)), and I'd put in the `test` because this is not for science analysis, but mainly for testing (and maybe for an example, because we don't have something better that's public.